specs/waku/v2/waku-filter.md

```diff
@@ -44,6 +55,21 @@ It is worth noting that a light node could get by with only using the `store`
 protocol to query for a recent time window, provided it is acceptable to do
 frequent polling.
 
+# Security Requirements
```

Comment:
We talked about this before, and it is a bit out of scope, but the most important "feature" by far is Best Effort Message Persistence, and the most important assumption is High Available. These are the two facts I personally care most about when it comes to store protocol. Maybe worth capturing in an issue?

Reply:
Wrt privacy, it seems like we should mention in a Note/security consideration section that you are linking yourself to some content topics which reduces your privacy to some extent. That's the most relevant privacy trade-off (for bandwidth) that people should be aware of.

Reply:
Sorry, the first comment was re store protocol not filter, confused PRs